File: 28_sum_all_numbers_in_range.js
// Sum all numbers in a range

function sumAll(arr){
    var start = Math.min(arr[0], arr[1]);
    var end = Math.max(arr[0], arr[1]);

    var total = 0;
    for (var i = start; i <= end; i++){
        total += i;
    }

    return total;
}


console.log(sumAll([1,4]));
console.log(sumAll([5,1]));
console.log(sumAll([100,0]))

File: 28_sum_all_numbers_in_range.py
# Sum all numbers in a range

def sumAll(arr):
    start = min(arr)
    end = max(arr)
    total = 0

    for i in range(start, end+1):
        total += i
    return total

# Another simple method

def sumAllSimple(arr):
    return sum(range(min(arr), max(arr)+1))

print(sumAll([1,4]))
print(sumAll([5,1]))
print("There is a simple way below..........")
print(sumAllSimple([1,4]))
print(sumAllSimple([5,1]))
print(sumAllSimple([0,100]))